docs/src/main/sphinx/connector/phoenix.rst

```diff
@@ -56,6 +56,10 @@ Property Name                                      Required   Description
                                                               default the location is ``/hbase``
 ``phoenix.config.resources``                       No         Comma-separated list of configuration files (e.g. ``hbase-site.xml``) to use for
                                                               connection properties.  These files must exist on the machines running Trino.
+``phoenix.max-scans-per-split``                    No         Maximum number of HBase scans allowed to be grouped into a single split. Default is 20.
+                                                              Lower values will lead to more splits in Trino.
+                                                              Can also be set via session propery ``phoenix.max_scans_per_split``.
+                                                              (This setting has no effect when guideposts are disabled in Phoenix.)
```

Comment:
Is this something that a Phoenix user will most likely know or should we link to something on Phoenix side?

Reply:
Good call... I'll link here: https://phoenix.apache.org/update_statistics.html